beetsplug/replaygain.py

```diff
@@ -1307,30 +1309,39 @@ def album_requires_gain(self, album):
                 (not item.rg_album_gain or not item.rg_album_peak)
                 for item in album.items()])
 
+    def _store(self, item):
+        try:
+            item.store()
+        except OperationalError:
+            # test_replaygain.py :memory: library can fail with
+            #    `sqlite3.OperationalError: no such table: items`
+            # but the second attempt succeeds
+            item.store()
```

Comment:
As far as I can tell, the exception is never thrown when working with an 'actual' database.